### DISCUSSION TITLE
Jellyfin debian error source

### DISCUSSION THREAD

**Discussion by eteixido1** [tteck/Proxmox#1463] [Help Needed] [answered]
Hello,
I don't know if this is the correct place but I can't install jellyfin with debian. I found an error in "jellyfin.sources" file because there are no version informed. The creation script die trying an apt update. I tryied to re-launch the script but it's to complex for me.

Kind regards,

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/1456

Advanced settings provide the option for users to switch between Debian and Ubuntu distributions. However, some applications or services may not be compatible with the selected distribution due to dependencies.

**Comment by tteck** [comment, marked as answer]
I have made a modification that potentially resolves the issue for Debian. https://github.com/tteck/Proxmox/commit/3f61e0739398de81e017ea7ba5e1a485b38e10b4

**Reply by eteixido1** [reply]
> I have made a modification that potentially resolves the issue for Debian. [3f61e07](https://github.com/tteck/Proxmox/commit/3f61e0739398de81e017ea7ba5e1a485b38e10b4)

Thank you tteck, now it works perfectly. Thank you again for your time.
